test/e2e/hazelcast_wan_test.go

```diff
@@ -3,10 +3,11 @@ package e2e
 import (
 	"context"
 	"fmt"
-	"github.com/hazelcast/hazelcast-platform-operator/internal/naming"
 	"strconv"
 	. "time"
 
+	"github.com/hazelcast/hazelcast-platform-operator/internal/naming"
```

Comment:
```suggestion
	n "github.com/hazelcast/hazelcast-platform-operator/internal/naming"
```

And also, could you move it to the last section?

```
	hazelcastcomv1alpha1 "github.com/hazelcast/hazelcast-platform-operator/api/v1alpha1"
	hzclient "github.com/hazelcast/hazelcast-platform-operator/internal/hazelcast-client"
	hazelcastconfig "github.com/hazelcast/hazelcast-platform-operator/test/e2e/config/hazelcast"
        n "github.com/hazelcast/hazelcast-platform-operator/internal/naming"
```

Reply:
At the repository, we imported `github.com/hazelcast/hazelcast-platform-operator/internal/naming` as `n` everywhere. That's why I requested it. I was aware that it would not compile, but my assumption was that related parts could be updated based on this change as well.